cli.js

```diff
@@ -90,6 +92,31 @@ const save = () => {
 
     // TODO handle creating files
     // TODO handle deleting files
+    const oldKeys = Object.keys(previousDocument);
+    const newKeys = Object.keys(currentDocument);
+    const removedKeys = oldKeys.filter((key) => !newKeys.includes(key));
+    removedKeys.forEach((key) => {
+      fs.unlinkSync(fullPath + '/' + previousDocument[key].name, (err) => {
+        process.on(err, () => { });
```

Comment:
This does not look like correct usage of `process.on`. See https://nodejs.org/api/process.html#process_process_events

I would recommend that we just throw the error to surface it.
```suggestion
        if (err) throw err;
```